src/main/java/fi/aalto/cs/apluscourses/intellij/activities/InitializationActivity.java

```diff
@@ -60,6 +62,16 @@ public void runActivity(@NotNull Project project) {
       notifier.notify(new NetworkErrorNotification(e), project);
       return;
     }
+
+    CoursePluginVersion minVersion = course.getRequiredPluginVersion();
+    CoursePluginVersion.Status versionCheckResult = minVersion.checkVersion();
```

Comment:
Nit: just noting that we have `var` now :wink:

Reply:
~~Finally, Java becomes almost as good as C#~~